recipes/gmap/2014.12.28/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: gmap
+  version: "2014.12.28"
+source:
+  url: http://research-pub.gene.com/gmap/src/gmap-gsnap-2014-12-28.tar.gz
+  sha1: fcd08a9ce1336c3935c5d43ab87967dbdfe53d94
+requirements:
+  build:
```

Comment:
this should be the host section imho

Reply:
Just added a host section for the latest commit